integration/tests/cook/util.py

```diff
@@ -100,21 +105,41 @@ def submit_job(cook_url, **kwargs):
     return job_spec['uuid'], resp
 
 
-def wait_for_job(cook_url, job_id, status, max_delay=120000):
+def load_job(cook_url, job_id):
```

Comment:
`get_job` is in `test_basic`. I have removed `get_job` and used `util.load_job` instead.